x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/roles/60_bulk_roles.yml

```diff
@@ -82,3 +82,17 @@ teardown:
           }
   - match: { deleted.0: "admin_role" }
   - match: { deleted.1: "role_with_description" }
+
+  # query match_all roles
+  - do:
```

Comment:
I think this has to happen before the bulk delete, now it happens after the roles were deleted and should fail.